README.md

```diff
@@ -820,6 +820,7 @@ API | Description | Auth | HTTPS | CORS |
 | [RandomUser](https://randomuser.me) | Generates random user data | No | Yes | Unknown |
 | [RoboHash](https://robohash.org/) | Generate random robot/alien avatars | No | Yes | Unknown |
 | [This Person Does not Exist](https://thispersondoesnotexist.com) | Generates real-life faces of people who do not exist | No | Yes | Unknown |
+| [UUID Generator](https://www.uuidtools.com/docs) | Generate UUIDs | No | Yes | No |
```

Comment:
The `uuidtools.com/docs` uses cloudflare service, this broken our tests.

The solution I see to solve this without having to change anything in your project is to add the repository link on github.

Can you do that, please?

```suggestion
| [UUID Generator](https://github.com/aarreedd/uuidtools.com) | Generate UUIDs | No | Yes | No |
```